packages/platform-android/src/utils/getAndroidProject.ts

```diff
@@ -0,0 +1,56 @@
+import {Config, AndroidProjectConfig} from '@react-native-community/cli-types';
```

Comment:
I refactored these functions from `platform-android/src/commands/runAndroid/index.ts` per @grabbou's suggestion to this new file. Those functions are hence used in both the above `downloadProfile.ts` file and the `runAndroid/index.ts` file.